@LinProp/dft.m

```diff
@@ -0,0 +1,44 @@
+function X = dft(A)
+% LinProp dft
+%   X = dft(A) computes the discret fourier transform.
+
+% Michael Wollensack METAS - 29.04.2022
+
+    numlib = NET.createGeneric('Metas.UncLib.LinProp.Ndims.ComplexUncNumLib', {'Metas.UncLib.LinProp.UncNumber'});
+    realTimeDomainData = ~A.IsComplex;
+    A = complex(A);
+    s = size(A);
+    am = Convert2UncArray(A);
```

Comment:
If you use `LinProp.Convert2UncArray(A);` instead of `Convert2UncArray(A);` you would not need a copy of `Convert2UncArray()` in this file. (The same applies to `Convert2LinProp` and `idft()`.)

However, as I do not have VNA Tools 2.5.4 I cannot fully test the functions.

Reply:
Thank you for reviewing this pull request. I merged this one it a little bit to early. So see https://github.com/wollmich/metas-unclib-matlab-wrapper/pull/56.

Reply:
No problem. I had actually forgotten to submit my review any only became aware of that when I got the notification about the merge.